src/render/mod.rs

```diff
@@ -538,13 +539,14 @@ impl Renderer {
     #[cfg(feature = "opengl")]
     pub(crate) fn new(
         builder: glutin::WindowBuilder,
-        context: glutin::ContextBuilder,
+        context: glutin::ContextBuilder<NotCurrent>,
         event_loop: &glutin::EventsLoop,
         source: &source::Set,
-    ) -> (Self, glutin::GlWindow, Factory) {
+    ) -> (Self, glutin::WindowedContext<PossiblyCurrent>, Factory) {
         use gfx::texture as t;
 
-        let (window, device, mut gl_factory, out_color, out_depth) = gfx_window_glutin::init(builder, context, event_loop).unwrap();
+        let (_windowedContext, device, mut gl_factory, out_color, out_depth) = gfx_window_glutin::init(builder, context, event_loop).unwrap();
+        let window = _windowedContext.window();
```

Comment:
Please don't use underscore in front of regular variables. `windowedContext` doesn't need it